src/plugins/GUITestBase/src/tests/common_scenarios/mca_editor/GTTestsMcaEditor.cpp

```diff
@@ -3770,6 +3771,31 @@ GUI_TEST_CLASS_DEFINITION(test_0041) {
     CHECK_SET_ERR("1036" == readLengthString, QString("15. Unexpected read length label: expected '%1', got '%2'").arg("1036").arg(readLengthString));
 }
 
+GUI_TEST_CLASS_DEFINITION(test_0042) {
+    // Open an MCA object.
+    GTFileDialog::openFile(os, testDir + "_common_data/sanger", "alignment_short.ugenedb");
```

Comment:
The file will be modified if you open it. It is not critical, if the test doesn't do any significant modifications, but it is better to make the test isolated. Copy the file to a sandbox and work with the copy.

Reply:
Our tests are safe for modifications. It is a part of the test suite logic: it runs rsync before each test to reset all data dirs.
Otherwise, any crashed test could affect all later ones.

BTW how re-positioning in MCA viewer can affect MCA data?

Reply:
Ok, rsync is enough.
Currently re-positioning can't affect MCA data, so I wrote that it is not critical. But for example we could add position saving to the database to open on the same position next time.